docker-compose.dev.yaml

```
@@ -76,10 +76,16 @@ services:
       minio:
         condition: service_started
     environment:
-      - MONGO_DB_URL=mongodb://mongo:27017/talawa-dev?replicaSet=rs0
+      - MONGO_DB_URL=mongodb://mongo:27017/talawa-api?replicaSet=rs0
       - NODE_ENV=development
       - REDIS_HOST=redis-stack-server
       - REDIS_PORT=6379
+      - ACCESS_TOKEN_SECRET=${ACCESS_TOKEN_SECRET}
+      - REFRESH_TOKEN_SECRET=${REFRESH_TOKEN_SECRET}
+      - MINIO_ROOT_USER=$(MINIO_ROOT_USER)
+      - MINIO_ROOT_PASSWORD=$(MINIO_ROOT_PASSWORD) 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)